### PR TITLE
Fixes #1157: Usage of += in strings inside loop

### LINKED ISSUE #1157: Usage of += in strings inside loop
**Issue**
As every String concatenation copies the whole String, usually it is preferable to replace it with explicit calls to StringBuilder.append() or StringBuffer.append()
In DAO.java += has been used in loop.  StringBuilder.append() or StringBuffer.append() can be used there.

**Would you like to work on the issue?**
Yes.

### PR DESCRIPTION
**Pull request by ByeongGil-Jung**
Fixes #1157

Changes: In DAO.java, Using StringBuilder.append() instead of '+='

### CODE REVIEW

**Comment by yashasvi128** [comment]
@ByeongGil-Jung I mentioned in the issue that I was going to work on this. But its okay. Next time please ask on the issue if you want to work on it.

**Comment by ByeongGil-Jung** [comment]
Ah, I'm really sorry..
I couldn't read the below sentence.
It's all my fault. I apologize to you. Sorry..

File: src/ai/susi/DAO.java [hunk 1]
```diff
@@ -889,23 +889,21 @@ public static void pushCommit(Git git, String commit_message, String userEmail)
...
+        StringBuilder conflictLines= new StringBuilder();
         try {
             result = IO.readFileCached(Paths.get(DAO.conf_dir + "/templates/conflicts-mail.txt"));
             for (String path :mergeResult.getConflicts().keySet()) {
                 int[][] c = mergeResult.getConflicts().get(path);
-                conflictLines+= "Conflicts in file " + path + "\n";
+                conflictLines.append("Conflicts in file ").append(path).append("\n");
                 for (int i = 0; i < c.length; ++i) {
-                    conflictLines+= "  Conflict #" + i+1 +"\n";
+                    conflictLines.append("  Conflict #").append(i).append(1).append("\n");
```
Comment (Orbiter) [inline src/ai/susi/DAO.java]: appending i and 1 is not the same as computing the sum of i and 1

Comment (ByeongGil-Jung) [reply src/ai/susi/DAO.java]: Thanks for letting me know !
I'll modify it ! :)

**Pull request closed without merging**